HealthcareSystem/PatientWebAppUnitTests/CreateObjectsForTests/CreatePatient.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Backend;
+using Model.Enums;
+using Model.Users;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PatientWebAppTests.CreateObjectsForTests
+{
+    public class CreatePatient : ICreateObject
+    {
+        public object createValidObject()
```

Comment:
Why **object**, maybe we should use the Patient return type. Do we start our methods with big letters or small? I forget, what is the convention? If we use one convention, then we should use that in all our code.

Reply:
I accidentally put the name of the method with small letter.

Reply:
It's okay, I just got confused and asked about the convention.